### DISCUSSION TITLE
mergeDeepWith

### DISCUSSION THREAD

**Discussion by filipw01** [remeda/remeda#949] [Suggest a Function]
What do you think about adding equivalent of https://docs-lodash.com/v4/merge-with/ ?

By implementing it we could also add `lodash.merge` to the list of one line replacements because current `mergeDeep` from `remeda` doesn't ignore `undefined` values when merging which could be easily customized with `mergeWith`

**Comment by cjquines** [comment]
i don't find this a super compelling reason to add mergeWith; in particular it means throwing away a lot of type information?

**Comment by filipw01** [comment]
Yes, that's why I didn't jump right into opening a PR
